examples/getting_started/react/App.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import { useOfflineMutation } from 'react-offix-hooks';
+import { CacheOperation } from 'offix-cache';
+import Modal from './components/Modal';
+import AddTodo from './components/AddTodo';
+import Loading from './components/Loading';
+import Error from './components/Error';
+import TodoList from './components/TodoList';
+import { GET_TODOS, ADD_TODO } from './gql/queries';
+
+const App = ({ client }) => {
+
+  const { loading, error, data } = useQuery(GET_TODOS);
+
+  if (loading) return <Loading />;
+
+  if (error) return <Error error={error} />;
+
+  const [initialized, setInitialized] = useState(false);
+  const [modalActive, setModalActive] = useState(false);
+  const [isOnline, setIsOnline] = useState(true);
+
+  useEffect(() => {
+    async () => {
+      // get network status
+      const offline = await client.networkStatus.isOffline();
+      // set network state
+      setIsOnline(!offline);
+    }
+    setInitialized(true);
+  }, []);
+
+  // set up network listener to 
+  // detect when app goes offline
+  client.networkStatus.onStatusChangeListener({
```

Comment:
[Optional] It will be preferable to set that when creating client. 
We want to show here good patterns for using offix etc.